plugins/obs-widget/back.js

```diff
@@ -0,0 +1,56 @@
+const {ipcMain} = require("electron");
+const http = require("http");
+const registerCallback = require("../../providers/song-info");
+
+let currentSongInfo;
+const data = {
+	player: {
+		hasSong: false,
+		isPaused: true,
+		seekbarCurrentPosition: 0,
+	},
+	track: {
+		author: "",
```

Comment:
Since you assign a list to it, shouldn't the initial value be an empty list?

```suggestion
		author: [],
```